src/enumGen.ts

```diff
@@ -11,8 +11,8 @@ function generateJSONDumpOfDefSystem( ){
     var Universe10={};
     universe10.types().forEach(x=>{
         var props={};
-        x.properties().forEach(y=>props[y.nameId()]={name:y.nameId()});
-        (<def.NodeClass>x).customProperties().forEach(y=>props[y.nameId()]={name:y.nameId()});
+        x.allProperties().forEach(y=>props[y.nameId()]={name:y.nameId()});
+        (<def.NodeClass>x).customProperties().forEach(y=>props[y.nameId()]={name:y.nameId(),range:y.range().nameId(),domain:y.domain().nameId()});
```

Comment:
Storing inherited properties may complicate fixing the parser after introducing changes into definition system.